.flowconfig

```diff
@@ -62,4 +62,4 @@ module.file_ext=.json
 module.file_ext=.jsx
 
 [version]
-^0.49.1
+^0.52.0
```

Comment:
We do not use the latest Flow intentionally (causes issues with other libs). We stick to whatever RN uses.